server/items/itemController.js

```diff
@@ -61,6 +63,26 @@ module.exports = {
         //success callback of loadAllItems in app.js
         res.json(items);
     });
+  },
+  removeItem: function(req, res){
+    console.log("we are trying to remove: ");
+
+    var itemName = req.body.item;
+    var itemLocation = req.body.LatLng;
+
+    var removeItem = Q.nbind(Item.remove, Item);
+    removeItem({itemName: itemName, itemLng: itemLocation.lng, itemLat: itemLocation.lat})
```

Comment:
could we just pass in itemLocation (from line 71) instead of the extracted lat/lng properties?